packages/rancher-webhook/generated-changes/patch/Chart.yaml.patch

```diff
@@ -4,7 +4,7 @@
    catalog.cattle.io/namespace: cattle-system
    catalog.cattle.io/os: linux
    catalog.cattle.io/release-name: rancher-webhook
-+  catalog.cattle.io/rancher-version: ">= 2.5.10 <2.5.99"
++  catalog.cattle.io/rancher-version: ">= 2.5.10-0 <2.5.99-0"
```

Comment:
This should happen in the upstream chart

Reply:
right, it should be in the upstream chart, and this was a quick fix last time we did last time. 

I will close this PR and move the fix to the upstream chart.

Reply:
This is 2.5.10, so we have until next week